tmt/steps/provision/podman.py

```diff
@@ -275,18 +275,27 @@ def options(cls, how: Optional[str] = None) -> List[tmt.options.ClickOptionDecor
         """ Prepare command line options for connect """
         return [
             click.option(
-                '-i', '--image', metavar='IMAGE',
-                help='Select image to use. Short name or complete url.'),
+                "-i",
+                "--image",
+                metavar="IMAGE",
+                help="Select image to use. Short name or complete url."),
             click.option(
-                '-c', '--container', metavar='NAME',
-                help='Name or id of an existing container to be used.'),
+                "-c",
+                "--container",
+                metavar="NAME",
+                help="Name or id of an existing container to be used."),
             click.option(
-                '-p', '--pull', 'force_pull', is_flag=True,
-                help='Force pulling a fresh container image.'),
+                "-p",
+                "--pull",
+                "force_pull",
+                is_flag=True,
+                help="Force pulling a fresh container image."),
             click.option(
-                '-u', '--user', metavar='USER',
-                help='User to use for all container operations.')
-            ] + super().options(how)
+                "-u",
+                "--user",
+                metavar="USER",
+                help="User to use for all container operations."),
+            *super().options(how)]
```

Comment:
I really dislike how the fix is buried under changes that are not related to it :/ Reformatting all options just to put the `super()` into the list, that seems like a way too much noise for such a trivial change :(

Reply:
I find it more readable this way, but can revert the formatting. It will pass afaik.

Reply:
I'm not against the reformatting per se, just against doing it there and under the hood of moving `*super()` into a list literal. Some time ago, the change of the line length was discussed, and while it was increased, tmt code base was not reformatted to make use of the new length, as the patch would introduce quite a bit of noise into `git blame`. This feels like the very same case. Please, feel free to bring it to Tuesday's hacking meeting, but I'd advise against reformatting for the sake of reformatting, especially when it clouds the actual change.

Reply:
Reformatting was a bad idea, thanks for keeping me honest. Changed back.